podcast/the-changelog-360.md

```diff
@@ -92,7 +92,7 @@ We had a bunch of things, like we had this interceptor plugin, which the two of
 
 **Abhinav Asthana:** Yup. And we tried a lot of different ways. We tried to bootstrap the project, and at that time I think Postman transitioned -- so we tried a whole bunch of ways; we tried donations, sponsorships, and... One thing I wanted to do was I didn't wanna go into raising money too early at all. That was a fact I learned; if you go into the market trying to raise money and you're still like "Okay, I have a prototype, and stuff" \[unintelligible 00:18:38.27\] too late, and it's very painful, and I didn't wanna do that. So I was just looking for ways to sustain ourselves, and the donation bit didn't work out, as well the sponsorship didn't work out... So we introduced this in-app purchase called Jetpacks at that time. The idea was that you buy this $10 in-app purchase and it'll give you access to automation capabilities that Postman has.
 
-The funny thing in adverts was that you have a postman going on a bicycle from door to door, delivering mail, and then you get this cool, new version of postman, with jetpacks, and he's flying around your town, delivering mail. That was a \[unintelligible 00:19:17.03\] That made a bit of money... And our grand plan was that we were gonna be maximum five people working on this thing, and let's figure out what are the most important problems we can solve. It was enough if you translated dollars to rupees in India; you can pay rent and eat. Like being ramen profitable, that's the term now, which I learned a little later.
+The funny thing in adverts was that you have a postman going on a bicycle from door to door, delivering mail, and then you get this cool, new version of postman, with jetpacks, and he's flying around your town, delivering mail. That was what gave Postman it's logo. That made a bit of money... And our grand plan was that we were gonna be maximum five people working on this thing, and let's figure out what are the most important problems we can solve. It was enough if you translated dollars to rupees in India; you can pay rent and eat. Like being ramen profitable, that's the term now, which I learned a little later.
```

Comment:
There's a couple of extra words in there, I think this update sounds right:
> That was the imagery we had which gave Postman its logo.

Reply:
Thanks for the suggestion. Updated.